go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/ProtonMail/go-crypto v0.0.0-20230923063757-afb1ddc0824c
 	github.com/YakDriver/regexache v0.23.0
 	github.com/aws/aws-sdk-go v1.49.2
-	github.com/aws/aws-sdk-go-v2 v1.24.0
+	github.com/aws/aws-sdk-go-v2 v1.24.1
```

Comment:
Version bump is performed by `go mod tidy`, because we use AWS Support SDK. AWS SDK Go v2 exposes a package for each of the services. Support service [requires](https://github.com/aws/aws-sdk-go-v2/blob/service/support/v1.19.6/service/support/go.mod) AWS SDK Go v2 v1.24.1.